lib/src/theme/macos_theme.dart

```diff
@@ -291,6 +301,35 @@ class MacosThemeData with Diagnosticable {
   /// The default style for [Tooltip]s below the overall [MacosTheme]
   final TooltipThemeData tooltipTheme;
 
+  /// The density value for specifying the compactness of various UI components.
+  ///
+  /// {@template flutter.material.themedata.visualDensity}
```

Comment:
Replace this whole documentation block (from `@template` to `@endTemplate`) to `{@macro flutter.material.themedata.visualDensity}`